openedx/core/djangoapps/credentials/management/commands/notify_credentials.py

```diff
@@ -236,8 +236,7 @@ def get_course_keys(self, courses):
             try:
                 course_keys.append(CourseKey.from_string(course_id))
             except InvalidKeyError:
-                log.fatal("%s is not a parseable CourseKey", course_id)
-                sys.exit(1)
+                log.error("%s is not a parseable CourseKey", course_id)
```

Comment:
You probably will also want to change the docstring to not document that errors are fatal.

But more importantly, why this change? This is parsing user input on the command line -- wouldn't we want to fail hard so they user knows to correct a typo or whatever?